src/actions.rs

```diff
@@ -182,8 +180,9 @@ impl Action {
         };
 
         match event {
-            TestEvent::SetMergeNeeded(value) => self.0.borrow_mut().merge_needed = value,
-            TestEvent::SetSplitNeeded(value) => self.0.borrow_mut().split_needed = value,
+            TestEvent::SetChurnNeeded(churn_type) => {
```

Comment:
I would call this local variable `churn_needed` for consistency.

Reply:
or value for consistency :-)